bids/layout/models.py

```diff
@@ -329,6 +329,22 @@ def get_image(self):
                              "NiBabel can read.".format(self.path))
 
 
+class BIDSJSONFile(BIDSFile):
+
+    __mapper_args__ = {
+        'polymorphic_identity': 'json_file'
+    }
+
+    def get_dict(self):
+        ''' Return the contents of the current file as a dictionary. '''
+        return json.loads(self.get_json())
```

Comment:
For paranoid me -- that is if the file contains a dictionary, which is typical in case of BIDS but might become not the case e.g. if file contains a list.  Might be worth asserting the type of the structure here

```suggestion
        d = json.loads(self.get_json())
        if not isinstance(d, dict):
            raise ValueError("File %s is a json containing %s, not a dict which was expected" % (self.path, type(d)))
        return d
```